sections/about/BoardSection.jsx

```diff
@@ -13,31 +15,72 @@ export const BoardSection = ({
       <div>
         <Title>{board}</Title>
         <p>{boardDescription}</p>
```

Comment:
Do you want to add the option of right aligned text for some of the other boards + background svgs added in? (wrap title and description with div) 
```
<Info>
title + description 
</Info> 
... 
(styled info component takes alignment prop to align left/right) 
```

Reply:
I can TODO this (background svg) and oh yeah I didn't see that the original design has left and right align ty.